src/components/BookItem.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const BookItem = () => (
```

Comment:
- [ ] Let's set up this **`BookItem`** component to take in book data as a prop and render the title and author. This way we set up the component to accept data from the parent component **(BookList).**

  ```
  const BookItem = (props) => {
    const { title, author } = props;

    return (
      <div>
        <p className="bookCategory">Action</p>
        <h2 className="BookTitle">{title}</h2>
        <span className="author">{author}</span>
        {' '}
        ...
    ```
  
  ```
  # BookList.js
  
  const BookList = () => (
    <ul>
      <BookItem title="Hunger Games" author="Suzanne Collins" />
      ...
  ```

Reply:
Done.Thanks for the example!